gen_boot_scr.sh

```diff
@@ -1,11 +1,4 @@
 #!/bin/sh
 mkimage -C none -A arm -T script -d boot.cmd boot/boot.scr
-mkimage -C none -A arm -T script -d boot-bb2x.cmd boot/variants/bittboy2x_v1/boot.scr
-mkimage -C none -A arm -T script -d boot-bb2x.cmd boot/variants/bittboy2x_v2/boot.scr
-mkimage -C none -A arm -T script -d boot-bb35.cmd boot/variants/bittboy3.5/boot.scr
-mkimage -C none -A arm -T script -d boot-pocketgo-v90-q90.cmd boot/variants/pocketgo_TE/boot.scr
-mkimage -C none -A arm -T script -d boot-pocketgo-v90-q90.cmd boot/variants/v90_q90/boot.scr
```

Comment:
maybe let us leave at least one (e.g. v90_q90) boot.scr configuration as example and move it to backup folder while genereting, if there arouse need for non-standard setenv combination

Reply:
everything is in uboot

Reply:
Yes but it is s more convenient to do changes from sdcard repo with ``mkimage`` , otherwise you have to set SDK->change code in uboot->upload here to make it work as you intended (I will asure you that there are not many ppl who are comfy with that procedure).

At least we should add info to README how to write settings to boot.scr.

Reply:
ok, I will add test-boot.cmd for testing purposes

Reply:
already done